soccer/gameplay/tactics/coordinated_pass.py

```diff
@@ -175,15 +189,37 @@ def use_chipper(self, value):
     def on_exit_running(self):
         self.remove_subbehavior('receiver')
 
-    def on_enter_kicking(self):
+    # Funciton to find if ball is moving towards receive point
+    def ball_heading_to_receive_point(self):
+        ball_velocity = main.ball().vel;
```

Comment:
Make sure to test this on the field. In the past with the old vision filter, there existed a problem where the ball velocity was in the opposite direction of the kick for the first couple frames causing things to fail

Reply:
That shouldn't be an issue assuming a sufficiently high value of MIN_BALL_MOVING_SPEED. Something to keep an eye on during the testing day, though.